src/styles/Modal.css

```diff
@@ -0,0 +1,13 @@
+.Modal {
+  position: fixed;
+  z-index: 100;
+  top: 0;
```

Comment:
I like that you thought ahead and made this a global style in case we need it again. Maybe we can use this to replace some alerts in the future 🤔